src/main/java/cn/edu/tsinghua/tsfile/timeseries/write/InternalRecordWriter.java

```diff
@@ -45,12 +51,21 @@ public InternalRecordWriter(TSFileConfig conf, TSFileIOWriter tsfileWriter, Writ
         this.writeSupport = writeSupport;
         this.schema = schema;
         this.primaryRowGroupSize = conf.groupSizeInByte;
+        this.pageSize = conf.pageSizeInByte;
         this.oneRowMaxSize = schema.getCurrentRowMaxSize();
+        assert primaryRowGroupSize > oneRowMaxSize;
```

Comment:
This will affect the iotdb, if the user sets the size error.

Reply:
I've convert assert to throwing exception. This error can't be solved by file-level.